types/canvasjs/index.d.ts

```diff
@@ -1,992 +1,1690 @@
-// Type definitions for CanvasJS v1.5.1
+// Type definitions for CanvasJS 1.9
 // Project: http://canvasjs.com/
-// Definitions by: Mark Overholt <https://github.com/mover5>
+// Definitions by: ShuYin Zhang  <https://github.com/brutalimp>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare namespace CanvasJS {
+export namespace CanvasJS {
```

Comment:
`export * as namespace Foo` and `export as name Foo` aren't valid syntax. Only `export as namespace Foo` is valid syntax. But in this case you shouldn't use `export` at all, since this package doesn't define a module but rather a global variable, right? `declare namespace Foo { ... }` defines a global variable `Foo` with the specified contents.